libseccomp/src/filter_context.rs

```diff
@@ -842,6 +1054,16 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter or  writing to the file fails,
     /// an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// # use std::io::{stdout, Error};
```

Comment:
Do we need `Error`? I don't see it used.

Reply:
@rusty-snake, I have removed it, Thank you for reviewing.